pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
     "eth-typing",
     "hypothesis",
     "pytest",
```

Comment:
should pytest(cov) even be here? :thinking:

Reply:
in theory, i think it's here because of `boa.test` module